### PR TITLE
Fix #593; allow multiple answers to polls

### LINKED ISSUE #593: Allow multiple answers to polls
**Is your feature request related to a problem? Please describe.**
The API now allows users to answer a poll with multiple options - up to all options! Beta currently only allows a single option.

**Describe the solution you'd like**
For Beta to allow up to `max_options` to be selected on a poll. During creation, add "max options" number input. When responding, allow that many options to be selected, and submit those in the POST body for the PUT call.

**Additional context**
* [When creating a poll](https://pnut.io/docs/api/resources/polls/lifecycle#post-polls), `max_options` can be set on the poll, from 1 to the total number of options.
* [When responding to a poll](https://pnut.io/docs/api/resources/polls/lifecycle#put-polls-id-response) use the new `PUT /polls/{id}/response` endpoint instead of `PUT /polls/{id}/response/{position}` endpoint. This new endpoint expects a POST body parameter of `positions`, which is a list of poll IDs.

### PR DESCRIPTION
**Pull request by 33mhz**
Also allow changing responses after initial response, until poll is closed.

This adds two new abilities: multiple answers, and changing answers before the poll ends.

Because `CustomCheckbox` expects a boolean, I didn't use it, I just copied its classes/elements into the Poll... So that could be refactored. But this is the concept of new functionality.

### CODE REVIEW

**Comment by 33mhz** [comment]
It looks like the Travis CI failed on some prep command, unrelated to the code.

File: src/components/organisms/Poll.vue [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-        </a>
+        <div v-if="votable" class="custom-control custom-checkbox">
+          <input
+            type="checkbox"
+            :id="`pos-` + option.position"
+            v-model="checkedPositions"
+            :value="option.position"
+            :disabled="disabledOption(option.position)"
+            class="custom-control-input"
+          />
```
[the remark this answers is not in the document]
Comment (33mhz) [reply src/components/organisms/Poll.vue:21]: This is not right - the inputs are given IDs, and the labels are next to them, with `for={the ID}`, just like `CustomCheckbox`.

[the remark this answers is not in the document]
Comment (33mhz) [reply src/components/organisms/Poll.vue:21]: Screenshot from 2020-08-15 08-16-03

**Review by sunya9 — APPROVED** [review]
thx!

**Pull request merged**